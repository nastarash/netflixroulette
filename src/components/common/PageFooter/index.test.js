import React from 'react';
import { PageFooter } from './index';

describe('<PageFooter />', () => {
  test('should be rendered correctly', () => {
    const footer = shallow(
      <PageFooter />
    );
    expect(footer).toMatchSnapshot();
  });
});
