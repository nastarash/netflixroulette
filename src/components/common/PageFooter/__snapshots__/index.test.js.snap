// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageFooter /> should be rendered correctly 1`] = `
<footer
  className="page-footer"
>
  <Logo />
</footer>
`;
